server/src/test/java/com/exasol/testsetupabstraction/server/JsonBJsonMapperTest.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.testsetupabstraction.server;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+class JsonBJsonMapperTest {
```

Comment:
This only tests the happy path. If coverage is not good enough you could add tests with invalid json